web/libs/editor/src/stores/AppStore.js

```diff
@@ -363,6 +363,30 @@ export default types
         });
       }
 
+      /**
+       * Hotkey for next annotation
```

Comment:
We should fix the comments and the hotkeys.addNamed function, as it doesn't move through the annotation list but through the task list